src/TableOfContents/index.tsx

```diff
@@ -313,11 +345,11 @@ function DefaultRowImpl<T extends TableOfContentsItem>({ item, isExpanded, toggl
           {item.meta && <span className="text-sm text-left text-gray font-medium">{item.meta}</span>}
           {loadingElem}
           {actionElem}
+          {item.iconPosition === 'right' && iconElem}
           {isGroup && (
-            <FAIcon
-              className="TableOfContentsItem__icon"
-              icon={['far', isExpanded ? 'chevron-down' : 'chevron-right']}
-            />
+            <div onClick={() => isGroupItem && toggleExpanded()} className="px-2">
```

Comment:
IMO a nicer way to express this would be `onClick={isGroupItem ? toggleExpanded : undefined}`. A tiny bit more efficient as well but mainly more expressive:

*If it's a group item then we handle the click like this. Otherwise we **don't** handle it at all.*

Reply:
Good point. Done